### ISSUE TITLE
How long does it take? (BERT Model Finetuning using Masked ML objective)

### ISSUE THREAD

**Issue by echan00** [huggingface/transformers#1272]
I am about to finetune a multilingual BERT model using English and Chinese text from the legal domain.  

My corpus is around 27GB, how long should I expect to train 3 epochs (default parameters) using a Google TPU?
